docs/badge.md

```diff
@@ -26,11 +26,14 @@ indicate the status of an item to the user.
 This type of badge shows when no `value` prop is provided. This form is
 effective for showing statuses.
 
+### withBadge Higher-Order Component
+The withBadge HOC allows you to easily add badges to icons and other components.
+
 ## Usage
 
 ```js
 import { Text, View } from 'react-native'
-import { Avatar, Badge } from 'react-native-elements'
+import { Avatar, Badge, Icon } from 'react-native-elements'
```

Comment:
The import here for `withBadge`

Reply:
I'm sorry for all the mistakes that slipped by. Thank you for catching them!

Reply:
It's no problem man! Don't feel bad about it. It's normal 👍